openconfig_pyang/plugins/util/html_emitter.py

```diff
@@ -146,7 +146,7 @@ def genStatementDoc(self, statement, ctx, level=1):
     (prefix, last) = yangpath.remove_last(pathstr)
     prefix_name = ht.add_tag("span", prefix + "/", {"class":"statement-path"})
     statement_name = prefix_name + ht.br(level,True) + statement.name
-    s_div += ht.h4(statement_name, {"class":"statement-name","id":node_to_id(statement)},level,True)
+    s_div += ht.h4(statement_name, {"class":"statement-name","id":statement.attrs['id']},level,True)
```

Comment:
nit: There should be spaces between the ":" following the key and the dictionary member.